content/release/1.3.0.md

```diff
@@ -19,24 +17,23 @@ aliases:
   limitations under the License. See accompanying LICENSE file.
 -->
 
-![glacier-haiku](releases/1.3.0.jpg)
+![Grand-Canyon](releases/1.3.0.jpg)
 
 Apache Ozone 1.3.0 is released with following features:
 
 - Erasure Coding - V1 support (Online reconstruction support)
-- Erasure Coding - offline recovery support (HDDS-6462)
 - Container balancer
-- Limit number of rocksdb instances to 1 per disk on high-density datanodes (HDDS-3630)
-- Multi-tenancy Support for Ozone (S3 multi-volume support)
+- Limit number of rocksdb instances to 1 per disk on high-density datanodes
+- Ozone S3 Multi-tenancy Support
 - S3 Grpc improvements for metadata path
-- Support filesystem optimization for existing installations not already using it
+- Filesystem Optimized and Object Store bucket layout types
 
 This is a generally available (GA) release.
 It represents a point of API stability and quality that we consider production-ready. 
 
 
-Image credit: [Glacier National Park 8][image] by [Tony Hisgett][author], [CC BY 2.0][cc] / Text added to original
+Image credit: [Grand Canyon National Park 1][image] by [Jim Frazee][author], [CC BY 2.0][cc] / Text added to original
 
-[image]: https://www.flickr.com/photos/hisgett/8051784706/
-[author]: https://www.flickr.com/people/37804979@N00
+[image]: https://www.flickr.com/photos/sarhounds/50193622148/
+[author]: https://www.flickr.com/people/sarhounds
 [cc]: https://creativecommons.org/licenses/by/2.0/deed.en
```

Comment:
The license for this image does not allow reuse, we need to [find another](https://www.flickr.com/search/?text=%22grand%20canyon%22&license=2%2C3%2C4%2C5%2C6%2C9) with more permissive license.

How about [this one](https://www.flickr.com/photos/geewhypics/50720710687)?

Reply:
Sure. Thanks @adoroszlai. I'll update this image.